### PR TITLE
Endurance downwelling spectral irradiance QARTOD tables

### PR DESCRIPTION
**Pull request by cwingard**
For the climatology tables I used the `spectra` parameter (array index from 0 to 6 corresponding to the 7 wavelengths) for the `zinp` parameter. Hopefully this, along with the array of arrays structure for the gross range tables will cover our needs for the SPKIR.

### CODE REVIEW

**Review by mvardaro — COMMENTED** [review]
Mostly looks good to me (although the array of parameters is hard to parse). One minor note that may or may not affect the Gross Range values: based on the documentation for the formerly-Satlantic-now-Sea-Bird-like-everything-else OCR-507 radiometer, the minimum valid irradiance value should be 0.0025, not zero. That said, looking at some of the recent cabled and uncabled SPKIR data, I'm seeing a lot of zeroes and even slightly negative values. So we should either amend the minimum fail range to the vendor specified value and potentially add a lot of fail flags (because the zeroes are not valid data) or keep the existing value and reduce but not eliminate the number of flags.

**Comment by cwingard** [comment]
@mvardaro I'm inclined to stick with the 0 for now (maybe even slightly negative?). I'd like to look into using a secondary, in-situ calibration offset to adjust the dark counts for each channel. If we flag all those near 0 values as fail, we might end up deleting valid measurements. All seven channels are slightly different during dark conditions (nighttime, no moon), which may be due to the electronics of our systems (much like WET Labs recommended re-measuring the dark counts for the FLORT after connecting it to a users system).

**Review by mvardaro — APPROVED** [review]
Fair enough. We should document that difference from the vendor specs, but otherwise this looks good.
